python/Makefile

```diff
@@ -0,0 +1,122 @@
+#
+# This Makefile serves as convenient command-line auto-completion when working on Python bindings
+#
+TOOLBOX_DIR?=../toolbox
+PROJECT_NAME=xnvme
+XNVME_CAPI_HEADERS=$(shell ls ../include/libxnvme* | grep -v util | grep -v libxnvmec)
+
+define default-help
+# invoke: 'make uninstall', 'make install'
+endef
+.PHONY: default
+default: uninstall install
+	@echo "## py: make default"
+	@echo "## py: make default [DONE]"
+
+define build-help
+# invoke: 'python3 setup.py build'
+endef
+.PHONY: build
+build:
+	@echo "## py: make build"
+	@python3 setup.py build
+	@echo "## py: make build [DONE]"
+
+define build-sdist-help
+# Create a source package (invoke: 'python3 setup.py sdist')
+endef
+.PHONY: build-sdist
+build-sdist:
+	@echo "## py: make build-sdist"
+	@python3 setup.py sdist
+	@echo "## py: make build-sdist [DONE]"
+
+define install-help
+# install for current user
+endef
+.PHONY: install
+install:
+	@echo "## py: make install"
+	@python3 -m pip install . --user
+	@echo "## py: make install [DONE]"
+
+define uninstall-help
+# uninstall
+#
+# Prefix with 'sudo' when uninstalling a system-wide installation
+endef
+.PHONY: uninstall
+uninstall:
+	@echo "## py: make uninstall"
+	@python3 -m pip uninstall ${PROJECT_NAME} --yes || echo "Cannot uninstall => That is OK"
+	@echo "## py: make uninstall [DONE]"
+
+define install-system-help
+# install system-wide
+#
+# install system-wide
+endef
+.PHONY: install-system
+install-system:
+	@echo "## py: make install-system"
+	@python3 -m pip install .
+	@echo "## py: make install-system [DONE]"
+
+define test-help
+# Run pytest on tests/
+endef
+.PHONY: test
+test:
+	@echo "## py: make test"
+	@python3 -m pytest --pyargs xnvme
```

Comment:
Omit `--pyargs`

Reply:
As stated elsewhere, it is there for a reason.